ur-packages/hex-string/src/HexString.ts

```diff
@@ -45,8 +45,8 @@ export class HexString extends RegistryItem {
     }
   }
 
-  public getData = () => this.data;
-  public getStringData = () => this.data.toString("hex");
+  public getData = () => this.data.toString("hex");
```

Comment:
Why is "getData" public if "toHex" returns the same?

Reply:
Actually the data was meant to be read as hex. 
So by default it should return hex encoded